user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+:ref:`Entities guide <user-guide-entity-management-from-UI>`. So, the system administrator can change the attachments 
+settings, define whether the entity should be displayed on a grid and/or a View page, whether it will be exported 
+to a .CSV file, and define other settings. For some of them it is also possible to add new fields, as described in the 
+:ref:`Entity Fields guide <user-guide-field-management>`. 
+
+These actions are only available if this capability is set to “System”, and are usually performed by the system 
+administrators.
+
+
+.. _admin-capabilities-org-calendar-events: 
+
+Manage organization calendar events
+-----------------------------------
+
+The capability enables users to create, edit and delete events in the organization-wide calendars, which are described 
+in more details in the :ref:`corresponding section <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+This capability can be utilized by the organization-level managers and HRs or other employees authorized to manage the 
+organization-wide calendars
+
+Users can view organization-wide calendars, add them to their own calendar views and copy related events to their own 
+calendars, regardless of this capability.
+
+
+.. _admin-capabilities-sys-calendar-events: 
+
+Manage system calendar events
+-----------------------------
+
+The capability enables users to create, edit and delete events in the system-wide calendars, which are described 
+in more details in the :ref:`corresponding section <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+This capability can be utilized by the company managers and HRs or other employees authorized to manage calendars at the 
+system-wide level.
+
+Users can view organization-wide calendars, add them to their own calendar views and copy related events to their own 
+calendars, regardless of this capability.
+
+
+.. _admin-capabilities-sys-calendars: 
+
+Manage system calendars
+-----------------------
+
+The capability enables users to :ref:`create <user-guide-calendars-system>` and 
+:ref:`manage <user-guide-calendars-manage>` system-wide calendars.
+
+It is usually available for system administrators and top managers.
+
+
+.. _admin-capabilities-passwords:
+ 
+Manage users' passwords
+-----------------------
+
+The capability enables users to change passwords of other users. Usually, it is only granted to the system 
+administrators, and can be used when :ref:`creating or editing a user record <user-management-users>`. 
+
+The capability does not influence the user's ability to 
+:ref:`edit own password <user-guide-getting-started-profile-password>` from the *"My User"* page.
+
+
+.. _admin-capabilities-merge:
+
+Merge entities
+--------------
+
+The capability enables users to :ref:`merge <user-guide-ui-components-grids-delete-merge>` several records of the same 
+entity. This is mostly applicable to the merge of :ref:`accounts <user-guide-accounts>`.
```

Comment:
Comment: In the second sentence, "the merge of accounts" is unclear, mostly because "merge" is being used as a noun, though it is a verb. ("Merger" is the noun form.) However, it's also unclear since the first sentence is saying that this capability is used to merge records, while the second is saying it is mostly used to merge accounts. Am I misinterpreting this? Can it be written more clearly some way?